### ISSUE TITLE
Remove synonym support (from 714)

### ISSUE THREAD

**Issue by gigamorph** [project-lux/lux-marklogic#21]
**Problem Description:**
Remove synonym support rather than restoring. This would include removing documentation, code,  sample thesaurus, and Gradle task related to synonym support. 

**Expected Behavior/Solution:**
- Remove synonym support.  Detailed in requirementes below

**Requirements:**
- Remove synonym support: 
- - documentation 
- - code
- - sample thesaurus
- - gradle task  `mlLoadData `

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

~~- [ ] Wireframe/Mockup - Heather~~
~~- [ ] Committee discussions - Sarah~~
- [ ] Feasibility/Team discussion - Sarah
- [ ] Backend requirements - TBD
- [ ] Frontend requirements- TBD
- [ ] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**
- N/A

**Dependencies/Blocks:**
N/A

**Related Github Issues:**
- https://git.yale.edu/lux-its/marklogic/pull/724

**Related links:**
N/A

**Wireframe/Mockup:**
N/A

**Comment by brent-hartwig** [comment]
@roamye, @prowns and @azaroth42, what are your thoughts on any LUX backend endpoint consumers wanted synonym support?  When this was enabled, I recall stakeholders being confused by some of the search results and that we were also interested in thesauri better geared towards LUX's data.  If we do not believe synonym support will ever be needed / re-enabled, we could delete some code.

cc: @clarkepeterf

**Comment by azaroth42** [comment]
I'm very doubtful that synonym support will be ever useful, given the wide range of subjects. Happy for code to go away, personally.

#agenda

**Comment by roamye** [comment]
since @azaroth42 won't be here this week. I have added this as a topic for the following week's agenda (8/23)

**Comment by roamye** [comment]
IT Team [Meeting 8/28](https://docs.google.com/document/d/1k8LKLtVtk-ZwJLkRYumSxB42_kcFp0hxRUn7yzIJiBA/edit?tab=t.0): 
Decision to not restore synonym support.

Should we remove code for synonym support from mark logic? #agenda    

@clarkepeterf @brent-hartwig

**Comment by brent-hartwig** [comment]
> Should we remove code for synonym support from mark logic?

Yes, please.  And the sample thesaurus and related documentation (e.g., the `mlLoadData` Gradle task is officially no longer needed by this project.).

**Comment by jffcamp** [comment]
We can get rid of documentation, sample thesaurus, and Gradle task in one ticket. Repurpose this ticket. Peter to update description.

**Comment by clarkepeterf** [comment]
Ticket is reformatted
